packages/clay-time-picker/src/index.tsx

```diff
@@ -303,7 +310,7 @@ const ClayTimePicker: React.FunctionComponent<IProps> = ({
 
 	return (
 		<div className="clay-time">
-			<ClayInput.Group>
+			<ClayInput.Group {...otherProps}>
```

Comment:
@bryceosterhaus I wasn't sure where to add this.
The `otherProps` in this case are used for `aria-*` attributes but it might be better to pick a better name?

Reply:
This is a bit confusing to have access to `aria-*`, for the context outside the component I think it makes sense, but adding it inside the component is a bit odd, depending on the context would probably be in the `inputs`?

Reply:
Yes, that's why I said I wasn't really sure of how to put them :joy: 

I agree that having them in the `input`s would be a good idea, but we'd need a way to separate what goes into each one, do you have an idea how we could do that and keep the API simple?

Reply:
No 😅, this is pretty confusing now haha

Reply:
@matuzalemsteles I'll make a few changes to this PR. 
For the moment I won't focus on accesibility related attributes (see [this](https://github.com/liferay/clay/issues/2458#issuecomment-532570784) comment) but just the `name` as this component will probably be used in forms.